Wikipedia/Localizations/shn.lproj/Localizable.strings

```diff
@@ -0,0 +1,376 @@
+// Messages for Shan (လိၵ်ႈတႆး)
+// Exported from translatewiki.net
+// Author: Saimawnkham
+
+"about-content-license" = "ဝႂ်ၶႂၢင်း ၶေႃႈၼႂ်း";
+"about-content-license-details" = "သင်ၸိူဝ်ႉဝႃႈ ဢမ်ႇပႆႇလႆႈမၵ်းမၼ်ႈဝႆႉၼႆ တူဝ်ၼႂ်းမၼ်း ၸၢင်ႈၵႂႃႇဢဝ်လႆႈ ၽၢႆႇတႂ်ႈ  $1 လႆႈယူႇ။";
```

Comment:
It's irrelevant at this point because iOS doesn't support shn - we can revisit if/when iOS supports shn (we'll see the change in the iOS Native Localization file)